app/models/data_export.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+# entity class for DataExport
+class DataExport < ApplicationRecord
+  has_logidze
+  acts_as_paranoid
+
+  belongs_to :user
+
+  has_one_attached :file
```

Comment:
**nitpick (blocking):** Could you add validation for `export_type` and `status` to only allow the types and status that should be supported.

Reply:
Added in b84d404d6c3b808fe8db407a866810630178247e